inscriptions/ISic020988.xml

```diff
@@ -0,0 +1,231 @@
+                            <supportDesc>
+                                <support>
+                                    <p>Fragment of wall and rim of a black-glazed fish-plate.</p>
+                                    <material ana="#material.clay.fired" ref="https://www.eagle-network.eu/voc/material/lod/131.html">ceramic</material>
+                                    <objectType ana="#object.vessel.plate" ref="http://kerameikos.org/id/plate">plate</objectType>
+                                    <dimensions>
+                                        <height unit="cm"/>
+                                        <dim type="diameter" unit="cm"/>
+                                        <width unit="cm"/>
+                                        <depth unit="cm"/>
+                                    </dimensions>
+                                </support>
+                                <condition ana="#condition.fragment"/>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout>
+                                    <rs ana="#execution.scratched" ref="https://www.eagle-network.eu/voc/writing/lod/9">scratched</rs>
+                                    <p>The text is placed just below the rim, on the outer surface of the plate.</p>
+                                    <damage ana="#text_condition.incomplete"/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote>
+                                <!--  -->
+                                <locus from="line1" to="line1">Line 1</locus>
+                                <dimensions type="letterHeight">
+                                    <height unit="mm"></height>
+                                </dimensions>
+                            </handNote>
+                        </handDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origPlace>
+                                <placeName type="ancient" ref="http://pleiades.stoa.org/places/462487">Segesta</placeName>		
+                                <placeName type="modern" ref="http://sws.geonames.org/2523171">Segesta</placeName>
+                                <geo>37.94171, 12.83649</geo>
+                            </origPlace>
+                            <origDate datingMethod="#julian" evidence="archaeological-context material-context lettering" notBefore-custom="-0500" notAfter-custom="-0400">500—400 BCE (Agostiniani 2021).</origDate>
+                        </origin>
+                        <provenance type="found">Found in the area of the stoa (Western wing)<geo></geo>.</provenance>
+                        <provenance type="rediscovered">Stored in the warehouse of the Parco Archeologico di Segesta<geo></geo>.</provenance>
```

Comment:
storage is not the same as rediscovered (see previous comments) - surely just 'recorded' or similar?